### ISSUE TITLE
metacarpal fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10702]
**Preferred term label:**
metacarpal fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Metacarpal fracture
Articular fracture of the base of the thumb metacarpal

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001126
- label: Metacarpal fracture
- parents: [Fractured hand bones (HP:0004277)](https://hpo.jax.org/app/browse/term/HP:0004277)
- pmid: 
- def: A partial or complete breakage of a metacarpal bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10702
- synonyms: Fractured metacarpal bone(NONE)

**Comment by MickeySegal** [comment]
What is the "NONE" in the synonyms?

**Comment by pnrobinson** [comment]
Let me check the NONE. It should be EXACT. Reopening until this is checked.

**Comment by pnrobinson** [comment]
@MickeySegal 
sorry, this is seems to be an error in the script I am using to summarize the new term definitions. NONE should be (and is, in the actual file) EXACT.
